### DISCUSSION TITLE
Global saved posts

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#112] [Practical Ideas]
Similar to global draft idea by Eric, it'd be great to have server wide saved post list of user in LHS. It can act as post bookmarking to save important posts and then user can go through them in a single page instead of wandering through channels

**Comment by esethna** [comment]
Yes! We really need to improve our cross-team functions and this is certainly in the plans to make saved posts and recent mentions across teams!

**Comment by zefhemel** [comment]
Completely on board with this, in fact just wanted to submit this idea myself. Global all the things!
